mainnet_v1/gw-mainnet_v1-config-readonly.toml

```
@@ -144,29 +38,17 @@ enable_debug_rpc = false
 check_mem_block_before_submit = false
 
 [block_producer.block_producer]
-registry_id = 2
+address_type = "Eth"
 address = '0xb8cde090e6a4741b6450308fad1dc338c53936a0'
 
-[block_producer.rollup_config_cell_dep]
-dep_type = 'code'
-
-[block_producer.rollup_config_cell_dep.out_point]
-tx_hash = '0xd891e0e8c4864c730cc4b96b450ed1bcaf42e74c9eb1a38fb193caaf490f9a3d'
-index = '0x0'
 [block_producer.challenger_config.rewards_receiver_lock]
 code_hash = '0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8'
 hash_type = 'type'
 args = '0x65c41e085ce5edfc7013ae7191635b3888f155c9'
 
-[block_producer.challenger_config.burn_lock]
-code_hash = '0x0000000000000000000000000000000000000000000000000000000000000000'
-hash_type = 'data'
-args = '0x'
-
 [mem_pool]
-execute_l2tx_max_cycles = 3000000000
-restore_path = '/mnt/mem_block'
-
+execute_l2tx_max_cycles = 500000000
```

Why change this value?

---

```suggestion
execute_l2tx_max_cycles = 3000000000
```

Good catch.
It should be 3000000000, the same value as Godwoken fullnode.